doc/source/ray-air/check-ingest.rst

```diff
@@ -83,20 +87,31 @@ Enabling Streaming Ingest
 
     You should use bulk ingest when:
 
-     * you have enough memory to fit data blocks in cluster object store;
-     * your preprocessing step is expensive per each epoch; and
-     * you want best performance when both or either the above conditions are met.
+    * you have enough memory to fit data blocks in cluster object store; or
+    * your preprocessing step is expensive to recompute on each epoch
 
 .. tabbed:: Streaming Ingest (experimental)
 
-    In streaming ingest mode, :py:func:`~ray.air.session.get_dataset_shard` returns a :py:class:`~ray.data.dataset_pipeline.DatasetPipeline` pipeline that
-    can be used to read data in a streaming way.
-    To enable streaming ingest, set ``use_stream_api=True`` in the dataset config.
+    In streaming ingest mode, instead of loading the entire dataset into the
+    Ray object store at once, AIR will load a fraction of the dataset at a
+    time. This can be desirable when the dataset is very large, and caching it
+    all at once would cause expensive disk spilling. The downside is that the
+    dataset will have to be recomputed on each epoch, which can add overhead if
+    the computation is expensive.
+
+    To enable this mode, use the :py:meth:`max_object_store_memory_fraction
+    <ray.air.config.DatasetConfig>` argument. This argument defaults to -1,
```

Comment:
After the dataset refactor, will the default value here be changed to just delegating to the Dataset optimizer?

Reply:
Yes.